src/main/java/com/scalar/database/transaction/consensuscommit/RecoveryHandler.java

```diff
@@ -51,6 +64,11 @@ public void recover(Selection selection, TransactionResult result) {
     }
   }
 
+  /**
+   * Rollback {@link DistributedStorage} to the specified {@link Snapshot}
```

Comment:
I think this description misleads.
Records on `DistributedStorage` will be rolled back by this method.
But `DisributedStorage` isn't rolled back.

Reply:
Right. Thank you!